includes/base/autoloader-base.php

```diff
@@ -0,0 +1,278 @@
+<?php
```

Comment:
Move all `classes_map` related code to the `Autoloader`.
the base class shouldn't know about it.

Reply:
No require base, and useless. look pm for more info.